test/test_preparation_and_conversion.py

```diff
@@ -37,6 +38,15 @@ def prepared_features_test(cutout):
     assert set(cutout.prepared_features) == set(cutout.data)
 
 
+def update_feature_test(cutout):
+    """atlite should be able to overwrite a feature."""
+    tmp = Path(mkstemp(suffix='.nc')[1])
+    tmp.unlink()
```

Comment:
This probably breaks on a windows installation. You are creating a new temp file for writing, dropping its file handle (effectively leaking it, since it is a lowlevel file handle the file will not be closed until shutting down the python interpreter), then unlinking it (re-creating the race-condition, which the doc-string of `mktemp` warns about), ie deleting an opened file, what windows screams about (maybe it does not since it's within the same process).

The only safe way i found is the one from
https://github.com/PyPSA/atlite/blob/09714d595fc09f2aa1ba79bdd9e60391c582e880/atlite/data.py#L211-L215

Here i would just use the `tmp_path` fixture: https://docs.pytest.org/en/latest/tmpdir.html#the-tmp-path-fixture